### DISCUSSION TITLE
Karazhan Lockout timer - change?

### DISCUSSION THREAD

**Discussion by XC-ahhaVA** [chromiecraft/chromiecraft#7043] [General]
TLDR: Consider changing Karazhan ID reset to 3 days, or adding an NPC that let's you reset ID

Currently, Karazhan has a 7-day lockout timer. This was present for the whole of TBC and TBC Classic.
However, now that it is no longer current content, it would make more sense to have the same lockout timer as Zul'Aman (ZA), ergo, 3 days.

Since ZA was brought forth as mostly a catch-up raid, together with the new badge loot that was introduced, it served the same purpose. It would make sense to introduce a similar change to Karazhan, thus enabling it to fill the catch-up raid role better for its gear ilvl bracket.

Alternatively, there could be an NPC appearing after killing Prince that would allow you to reduce the lockout timer to just 12 hours. With the given restrictions or precautions to make it a one- or two-time-only use per week. Or he could be placed outside, and have a weekly quest to kill Prince, and only on completion allowing the reset.

This would help with gearing both new players on the server and alts for people who wish to experience other classes or roles.

**Comment by heyitsbench** [comment]
Seeing as it's not Blizzlike whatsoever for any version of the game, I am personally not in favor of this change.
